UsageControlFramework/src/test/java/it/cnr/iit/usagecontrolframework/rest/jgiven/stages/GivenMessage.java

```diff
@@ -98,8 +97,8 @@ public void tear() {
         return self();
     }
 
-    public GivenMessage a_policy_that_will_trigger_a_Deny_response() {
-        policy = readResourceFileAsString( testContext.getPolicyFileDeny() );
+    public GivenMessage a_policy_that_will_trigger_a_Deny_response() throws URISyntaxException, IOException {
```

Comment:
please remove as per my above comment and handle accordingly

Reply:
done